<ui version="4.0" >
 <class>Tools</class>
 <widget class="QWidget" name="Tools" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Tools</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeView" name="toolsListView" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights" >
      <bool>true</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="KPushButton" name="addButton" >
       <property name="text" >
        <string>Add</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="stdItem" stdset="0" >
        <number>27</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="propertiesButton" >
       <property name="text" >
        <string>Properties</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="stdItem" stdset="0" >
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="removeButton" >
       <property name="text" >
        <string>Remove</string>
       </property>
       <property name="stdItem" stdset="0" >
        <number>28</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kpushbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>Tools</receiver>
   <slot>addClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>propertiesButton</sender>
   <signal>clicked()</signal>
   <receiver>Tools</receiver>
   <slot>propertiesClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>Tools</receiver>
   <slot>removeClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
